sys/linux/socket_netlink_route_sched.txt

```diff
@@ -198,6 +724,113 @@ m_connmark_options [
 	TCA_CONNMARK_PARMS	nlattr[TCA_CONNMARK_PARMS, tc_connmark]
 ] [varlen]
 
+m_csum_options [
+	TCA_CSUM_PARMS	nlattr[TCA_CSUM_PARMS, tc_csum]
+] [varlen]
+
+m_gact_options [
+	TCA_GACT_PARMS	nlattr[TCA_GACT_PARMS, tc_gen]
+	TCA_GACT_PROB	nlattr[TCA_GACT_PROB, tc_gact_p]
+] [varlen]
+
+m_ife_options [
+	TCA_IFE_PARMS	nlattr[TCA_IFE_PARMS, tc_ife]
+	TCA_IFE_DMAC	nlattr[TCA_IFE_DMAC, mac_addr]
+	TCA_IFE_SMAC	nlattr[TCA_IFE_SMAC, mac_addr]
+	TCA_IFE_TYPE	nlattr[TCA_IFE_TYPE, int16]
+	TCA_IFE_METALST	nlattr[TCA_IFE_METALST, tca_ife_meta_policy]
+] [varlen]
+
+# TODO: the PAYLOAD could also be NULL
```

Comment:
What is PAYLOAD?

Reply:
The nlattr[] PAYLOAD. For example, IFE_META_SKBMARK could be nlattr[IFE_META_SKBMARK, int32] or nlattr[IFE_META_SKBMARK, void]

Reply:
I've just added optional template that can be used for this:
https://github.com/google/syzkaller/commit/031c7941d7bd97b4acbf9bcd2513ed695b387ff2
https://github.com/google/syzkaller/commit/be1a56115bbc3dac09ccb92162a8d5f8937ba492

I.e. you can do:
nlattr[IFE_META_SKBMARK, optional[int32]]

Reply:
Cool. Thanks for the work.